src/nvim/testdir/Makefile

```diff
@@ -61,6 +61,7 @@ NEW_TESTS ?= \
 	    test_timers.res \
 	    test_undo.res \
 	    test_usercommands.res \
+	    test_utf8.res \
```

Comment:
I noticed recently, if I'm not mistaken this will cause the test to run twice, since it's also listed in test_alot.vim.

On the other hand, if tests aren't listed in `NEW_TESTS` then `TEST_FILE=test_foo.res make oldtest` won't find the test. Which doesn't seem terribly important to me, since `test_alot.vim` is supposed to be fast anyway.